### ISSUE TITLE
Add emulated Tab, Enter, and Arrow keys for "allowAdvancedBindings"

### ISSUE THREAD

**Issue by ZwipZwapZapony** [jrbudda/Vivecraft_114#41]
Since version 1.14+ (I'm not quite sure when, but at least in 1.14.4) of vanilla Minecraft, it's possible to use Tab to highlight menu options, Enter to "click" on them, and the left and right arrow keys to move sliders around. Vivecraft (1.14.4 v3 r0) currently does not allow you to emulate those keys, which leaves some vanilla Minecraft "controls" out of Vivecraft.

I understand that you can just point and click at menu options with a VR controller... but it's worth keeping in mind that SteamVR Input (probably) allows traditional gamepad input, which don't have a "pointing" option, so while the userbase would of course be extremely small (if not non-existant), it would be nice for those users to be able to assign vanilla menu-navigation buttons to a gamepad.

It might even be useful for actual VR users that just have shaky hands, as then they could use buttons to more precisely slide sliders around to specific values without their hand shake affecting anything, although I wouldn't suggest making these controls part of any default VR controller configuration, but still.

~

TL;DR: Minecraft uses Tab, Enter, and the left/right arrow keys in menus. Vivecraft does not emulate this. I suggest allowing experts to configure these options when "allowAdvancedBindings" is enabled.

**Comment by baxcat** [comment]
This isn't so much an issue as it is a request for a feature that doesn't seem very useful.

> It might even be useful for actual VR users that just have shaky hands, as then they could use buttons to more precisely slide sliders around to specific values without their hand shake affecting anything

ehhh, that's pretty niche imo and it's pretty difficult to play most any VR game if you have notoriously shaky hands anyhow, idk. To me this seems like something that would only be worth putting time into if it was heavily requested or the devs just had nothing better to do on the mod

**Comment by ZwipZwapZapony** [comment]
> This isn't so much an issue as it is a request for a feature that doesn't seem very useful. -

I won't pretend that it's "not very useful", but it is a vanilla Minecraft control thing that doesn't exist in Vivecraft's SteamVR Input bindings. But yes, you are correct in it being a suggestion, not an issue... but this is still the best place to make suggestions, and at least one person (who I'm quite confident is even myself) on Vivecraft's Discord server has been told by one of Vivecraft's main developers to make suggestions as GitHub issues.
